src/physics/physicsmodel.cxx

```diff
@@ -28,7 +28,11 @@
  *
  **************************************************************************/
 
+#define NO_GLOBAL_NAMESPACE
```

Comment:
Is this needed here? Later we do `bout::globals::mesh` anyway

Reply:
Sorry, badly named macro. Defining the macro means we have to write out `bout::globals::mesh` in full, see comment above about `using namespace bout::globals`. I'll rename it, thinking of changing to `BOUT_NO_USING_NAMESPACE_BOUTGLOBALS`.